database/seedcsv.js

```diff
@@ -0,0 +1,220 @@
+      JSON.stringify([...new Set(xMContent)])}\',\'${
+      JSON.stringify([...new Set(xOs)])}\',${proc[Math.floor(Math.random() * proc.length)]},${
+      mem[Math.floor(Math.random() * mem.length)]},${
+      gphx[Math.floor(Math.random() * gphx.length)]},\'${
+      JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${
+      JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${
+      JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${JSON.stringify([
+      faker.lorem.word(),
+      faker.image.image(),
+      faker.lorem.word(),
+      faker.image.image(),
+      faker.lorem.word(),
+      faker.image.image(),
+      faker.lorem.word(),
+      faker.image.image(),
+    ])}\',\'${JSON.stringify([faker.lorem.word(), faker.image.image()])}\',\'${JSON.stringify([
+      faker.lorem.word(),
+      faker.image.image(),
+      faker.lorem.word(),
+      faker.image.image(),
+      faker.lorem.word(),
+      faker.image.image(),
+    ])}\',${xLanguages},\'${JSON.stringify([...new Set(xAchieves)])}',${Math.floor(Math.random() * 101)}\',\'${
+      JSON.stringify([...new Set(xGenres)])}\',${faker.company.companyName()},${faker.company.companyName()},${
+      faker.company.companyName()},${faker.date.past()},${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\',${
+      faker.commerce.productName()},${faker.image.image()},${faker.commerce.price()},${
+      faker.date.past()},${faker.image.image()},${reviewRatings[Math.floor(Math.random() * reviewRatings.length)]},${
+      Math.floor(Math.random() * 1001)},\'${JSON.stringify([...new Set(xTags)])}\'\n`;
+    try {
+      fs.appendFileSync('data.csv', data);
```

Comment:
how were you able to do this without creating a stream?

Reply:
It's a synchronous command. Just executes the write on one row at a time and waits. Slow but it gets the job done. I want to convert to the stream though.

Reply:
ah I see, thank you Jay!